server/lib/cache/redis.js

```diff
@@ -1,22 +1,20 @@
-import Promise from 'bluebird';
 import debug from 'debug';
 import redis from 'redis';
 
-const asyncRedis = Promise.promisifyAll(redis);
```

Comment:
We're still using Redis 3, which doesn't seem to be async by default:
https://github.com/redis/node-redis/blob/master/docs/v3-to-v4.md